src/token/types/unscoped.rs

```diff
@@ -25,22 +28,71 @@ use crate::token::{
     types::Token,
 };
 
-#[derive(Builder, Clone, Debug, Default, PartialEq, Serialize)]
+#[derive(Builder, Clone, Debug, Default, PartialEq, Serialize, Validate)]
 #[builder(setter(into))]
 pub struct UnscopedPayload {
+    #[validate(length(min = 1, max = 64))]
     pub user_id: String,
+
     #[builder(default, setter(name = _methods))]
+    #[validate(length(min = 1))]
     pub methods: Vec<String>,
+
     #[builder(default, setter(name = _audit_ids))]
+    #[validate(custom(function = "validate_audit_ids"))]
     pub audit_ids: Vec<String>,
+
+    #[validate(custom(function = "validate_future_datetime"))]
```

Comment:
we can't enforce this. The token passed by the user is parsed into this structure, and the token payload itself is valid. The resulting token would eventually be considered as expired, but it is still valid from the data point of view. Please remove any datetime validations.